sections/CampaignTimer/CampaignTime.Bushido.tsx

```diff
@@ -0,0 +1,61 @@
+export interface MyInterface {
+  textCampaign: string;
+  textButton: string;
+  textLink: string;
+  hours: number;
+  minutes: number;
+  seconds: number;
+}
```

Comment:
Todas as suas Props são obrigatórias, ou seja, o componente não funcionaria sem especifica-las. 

Idealmente o componente deveria ter o maior numero de props opcionais possíveis e valores default para elas.
```suggestion
export interface MyInterface {
  textCampaign?: string;
  textButton?: string;
  textLink?: string;
  hours: number;
  minutes: number;
  seconds: number;
}
```